docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -78,45 +78,85 @@ These instructions assume you have experience using the command line, understand
+   "dev-server": "$SERVER_START$"
+   ```
+
+1. Replace `$MANIFEST_FILE$` with the correct file name and folder location of your manifest.xml file.
+1. Replace `$SERVER_START$` with the command to start your web server. Later in these steps, the office-addin-debugging package will specifically look for the `dev-server` script to launch your web server.
+1. Save and close the `package.json` file.
+
+### Configure launch.json file
+
 1. Create a file named `launch.json` in the `\.vscode` folder of the project if there isn't one there already.
-1. Ensure that the file has a `configurations` array. The following is a simple example of a `launch.json`.
+1. Copy the following JSON into the file.
 
    ```json
    {
      // Other properties may be here.
-   
      "configurations": [
-   
-       // Configuration objects may be here.
-   
+       {
+         "name": "$HOST$ Desktop (Edge Chromium)",
+         "type": "msedge",
+         "request": "attach",
+         "useWebView": true,
+         "port": 9229,
+         "timeout": 600000,
+         "webRoot": "${workspaceRoot}",
+         "preLaunchTask": "Debug: Excel Desktop"
+       }
      ]
-   
      // Other properties may be here.
    }
    ```
 
-1. Add the following object to the `configurations` array.
+   > [!NOTE]
+   > If you already have a launch.json file, just add the single configuration to the `configurations` section.
+
+1. Replace the placeholder `$HOST$` with the name of the Office application that the add-in runs in; for example, `Outlook` or `Word`.
+1. Save and close the file.
+
+### Configure tasks.json
+
+1. Create a file named `tasks.json` in the `\.vscode` folder of the project.
+1. Copy the following JSON into the file. It contains a task that will start debugging for your add-in.
 
    ```json
    {
-      "name": "$HOST$ Desktop (Edge Chromium)",
-      "type": "pwa-msedge",
-      "request": "attach",
-      "useWebView": true,
-      "port": 9229,
-      "timeout": 600000,
-      "webRoot": "${workspaceRoot}",
-      "preLaunchTask": "Debug: Excel Desktop",
-      "postDebugTask": "Stop Debug"
-   },
+     "version": "2.0.0",
+     "tasks": [
+       {
+         "label": "Debug: $HOST$ Desktop",
+         "type": "shell",
+         "command": "npm",
+         "args": ["run", "start:desktop", "--", "--app", "$HOST$"],
+         "presentation": {
+           "clear": true,
+           "panel": "dedicated"
+         },
+         "problemMatcher": []
+       }
+     ]
+   }
    ```
 
-1. Replace the placeholder `$HOST$` with the name of the Office application that the add-in runs in; for example, `Outlook` or `Word`.
-1. Save and close the file.
+   > [!NOTE]
+   > If you already have a tasks.json file, just add the single task to the `tasks` section.
+
+1. Replace both instances of the placeholder `$HOST$` with the name of the Office application that the add-in runs in; for example, `Outlook` or `Word`.
```

Comment:
That semicolon pattern Linda called out is here too. Make sure whatever changes you make are propagated.